netbox/ipam/tables/ip.py

```diff
@@ -318,7 +318,8 @@ class IPAddressTable(BaseTable):
         verbose_name='NAT (Inside)'
     )
     assigned = BooleanColumn(
-        accessor='assigned_object_id',
+        linkify=True,
```

Comment:
change order of linkify and accessor so the variables will be sorted by alphabet order